app/controllers/chargeback_controller.rb

```diff
@@ -288,28 +288,28 @@ def cb_assign_field_changed
 
   # Add a new tier at the end
   def cb_tier_add
-    detail_index = params[:detail_index]
+    index = params[:index]
```

Comment:
@amaurygonzalez 
can there be just small change yet? 
    `raise _("Invalid input") unless is_integer?(params[:index])`
this is what @skateman  had on his mind.